src/mina-signer/consistency.unit-test.ts

```diff
@@ -178,6 +193,49 @@ let fullCommitment = hashWithPrefix(prefixes.accountUpdateCons, [
 ]);
 expect(fullCommitment).toEqual(ocamlCommitments.fullCommitment.toBigInt());
 
+// private key to/from base58
```

Comment:
It would be nice to serialize these tests to JSON or some format and have this code deserialise them and perform the tests.  This way other signer implementations can use them for tests also.  But that seems like a lot of extra work and it's a good first step to have tests in the first place.  So I suggest this just to note the idea for future.

Reply:
that's a good point! I made an issue: https://github.com/o1-labs/snarkyjs/issues/634
It shouldn't be that hard, let's revisit when (if) we implement chunked signatures in Rust?

Reply:
deal